src/map/skill.cpp

```diff
@@ -15571,7 +15572,6 @@ bool skill_check_condition_castbegin(struct map_session_data* sd, uint16 skill_i
 				break;
 			if (map_getcell(sd->bl.m,sd->bl.x,sd->bl.y,CELL_CHKWATER) && !map_getcell(sd->bl.m,sd->bl.x,sd->bl.y,CELL_CHKLANDPROTECTOR))
 				break;
-			clif_skill_fail(sd,skill_id,USESKILL_FAIL_LEVEL,0);
```

Comment:
Undocumented change?

Reply:
In aegis this never shows "skill failed" messages.

Reply:
It's strange I've never heard of the requirement, that CALL SPIRIT requires to be standing on water?

Reply:
? I think you're mixing things, this isn't related to MO_CALLSPIRITS, but AL_HOLYWATER and WZ_WATERBALL

Reply:
You removed this code portion from MO_CALLSPRITIS and not WZ_WATERBALL.
That's why I do not understand this. :(

Reply:
No, this is case ST_WATER ._.

Reply:
Ah damn github tricked me :(
Sorry totally my fault here!